### ISSUE TITLE
YAML output isn't useful

### ISSUE THREAD

**Issue by rossburton** [getpatchwork/git-pw#70]
For example:

```
$ git-pw series show 17817 -fyaml
- property: ID
  value: 17817
- property: Date
  value: '2023-10-16T19:44:55'
- property: Name
  value: 'patchtest: add to oe-core'
- property: URL
  value: https://patchwork.yoctoproject.org/project/oe-core/list/?series=17817
- property: Submitter
  value: Trevor Gamblin (tgamblin@baylibre.com)
- property: Project
  value: OpenEmbedded Core Layer
- property: Version
  value: 1
- property: Received
  value: 4 of 4
- property: Complete
  value: true
- property: Cover
  value: '717  [0/4] patchtest: add to oe-core'
- property: Patches
  value: '32402 [1/4] patchtest/requirements.txt: update'
- property: ''
  value: '32405 [2/4] patchtest: add supporting modules'
- property: ''
  value: '32404 [3/4] patchtest: add scripts to oe-core'
- property: ''
  value: '32403 [4/4] patchtest: set default repo and testdir targets'
```

The patches are not expressed in yaml in a way that is actually parsable.

**Comment by AnakinSolo1983** [comment]
Hello, Ross, this is Oleg, and I would like to work on the YAML issue. I am new to GitHub, so I don't know how to get assigned to an issue, and so I ask you if I could work on it.

Thank you.

**Comment by AnakinSolo1983** [comment]
> Feel free to work on it Oleg, I haven't done anything on this.

Thank you. But what about being assigned to it, I mean on the assignee menu?
It's just I really have little experience here in Github.

**Comment by stephenfin** [comment]
> The maintainer can do that, but that isn't essential.

Yes, not essential but done. Please let us know if you need any guidance. The developer docs should (hopefully) be relatively comprehensive.

**Comment by AnakinSolo1983** [comment]
Hello, Ross. I have fixed the problem and I am trying to push, but I have encountered an issue when running the command git push:
Pushing to https://github.com/getpatchwork/git-pw
Username for 'https://github.com': AnakinSolo1983
Password for 'https://AnakinSolo1983@github.com': 
remote: Permission to getpatchwork/git-pw.git denied to AnakinSolo1983.
fatal: unable to access 'https://github.com/getpatchwork/git-pw/': The requested URL returned error: 403

Can you please tell me why is this happening?
Thank you.

**Comment by AnakinSolo1983** [comment]
Thanks for the help. The pull request is submitted.
